ir/src/transition_constraints/graph.rs

```diff
@@ -125,21 +231,30 @@ impl Node {
 #[derive(Debug, Eq, PartialEq)]
 pub enum Operation {
     Constant(u64),
-    /// An identifier for a specific column in the current row in the main trace. The inner value is
-    /// the index of the column within the trace.
+    /// An identifier for a for a cell in the specified column in the current row in the main trace.
+    /// The inner value is the index of the column within the trace.
     MainTraceCurrentRow(usize),
-    /// An identifier for a specific column in the next row in the main trace. The inner value is
-    /// the index of the column within the trace.
+    /// An identifier for a cell in the specified column in the next row in the main trace. The
+    /// inner value is the index of the column within the trace.
     MainTraceNextRow(usize),
+    /// An identifier for a cell in the specified column in the current row in the auxiliary trace.
+    /// The inner value is the index of the column within the trace.
+    AuxTraceCurrentRow(usize),
+    /// An identifier for a cell in the specified column in the next row in the auxiliary trace. The
+    /// inner value is the index of the column within the trace.
+    AuxTraceNextRow(usize),
+    /// An identifier for a periodic value from a specified periodic column. The inner value is the
+    /// index of the periodic column within the declared periodic columns. The periodic value made
+    /// available from the specified column is based on the current row of the trace.
+    PeriodicColumn(usize),
```

Comment:
Would it make sense to track both the index as well as the cycle length for a periodic column here? This might make degree computation a bit more straight-forward.

Reply:
Yes, that's better, thanks